client/src/components/Comparison/Comparison.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
-import comparison from './Comparison.module.css';
-import Card from './Card/Card.jsx';
+// eslint-disable-next-line import/extensions
```

Comment:
what are these comments for?

Reply:
I will add a state to this component later. Now just disable the lint warning for this  stateless-function check